deploy/kubernetes-1.20/direct/kustomization.yaml

```diff
@@ -0,0 +1 @@
+resources: [ pmem-csi.yaml ]
```

Comment:
The deploy changes don't belong into the first commit, do they?

Reply:
> The deploy changes don't belong into the first commit, do they?
> 
> Or at least not according to the commit message ("changes to install Kuberntes v1.20 based test cluster") - it's about installing Kubernetes, not PMEM-CSI.

Okay, I split the commit into 2; manifest generation and cluster changes.